### PR TITLE
Add a simple type definition

### PR DESCRIPTION
**Pull request by u3u**

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { IntrinsicElementAttributes } from 'vue-tsx-support/types/dom'
+
+declare module 'vue-emotion' {
+  const styled: IntrinsicElementAttributes
+  export default styled
+  export const css: any
+  export const injectGlobal: any
+}
```
Comment (egoist) [inline types/index.d.ts]: 1. you can `export * from 'emotion'` here too
2. seems no need to wrap inside `declare module 'vue-emotion'`
3. `styled` does not always support `.div` `.a` methods unless you're using babel, not sure if we should include a definition for this. And I don't think `IntrinsicElementAttributes` is the correct type 🤔

Comment (u3u) [reply types/index.d.ts]: Yes, we should make a change based on https://github.com/emotion-js/emotion/blob/master/packages/react-emotion/types/index.d.ts

**Comment by u3u** [comment]
https://github.com/emotion-js/emotion/blob/88a18e51f60e8d4fcbb49a21b4d8a7a1a3153b32/packages/react-emotion/types/index.d.ts#L25-L27

https://github.com/emotion-js/emotion/blob/88a18e51f60e8d4fcbb49a21b4d8a7a1a3153b32/packages/react-emotion/types/index.d.ts#L37-L52

https://github.com/emotion-js/emotion/blob/88a18e51f60e8d4fcbb49a21b4d8a7a1a3153b32/packages/react-emotion/types/index.d.ts#L54-L56

We can replace `JSX.IntrinsicElements` with `IntrinsicElementAttributes`
But what about `StatelessComponent`, `ComponentClass`, `CSSProperties`?

**Comment by u3u** [comment]
For the time being, let's export everything from emotion first. What do you think?

**Comment by egoist** [comment]
I can merge this once the type for `styled` is defined 😄

**Comment by u3u** [comment]
Reopen a new branch

**Pull request closed without merging**